docker/ansible-in-docker/README.md

```diff
@@ -0,0 +1,26 @@
+ansible-in-docker
```

Comment:
Our pattern for directory names is to have it be the same as the container name.

So in this case `oso-ansible`